draft-ietf-quic-multipath.md

```diff
@@ -1433,41 +1467,61 @@ Path Identifier:
 means the current Connection ID can only be used on the corresponding path.
 
 Sequence Number:
-The sequence number assigned to the connection ID by the sender on the path 
-specified by Path Identifier, encoded as a variable-length integer. 
+The sequence number assigned to the connection ID by the sender on the path
+specified by Path Identifier, encoded as a variable-length integer.
 
 
-## MAX_PATHS frames {#max-paths-frame}
+## MAX_CLIENT_PATHS and MAX_SERVER_PATHS frames {#max-paths-frame}
 
-A MAX_PATHS frame (type=0x15228c0b) informs the peer of the cumulative number of paths 
-it is permitted to open. 
+The MAX_CLIENT_PATHS (type=0x15228c0c) and MAX_SERVER_PATHS (type=0x15228c0d) frames inform the peer of the cumulative
+number of client initiated and server initiated paths for which it MAY publish
+MP_NEW_CONNECTION_ID frames.
 
 MAX_PATHS frames are formatted as shown in {{fig-max-paths-frame-format}}.
 
 ~~~
-MAX_PATHS Frame {
-  Type (i) = 0x15228c0b,
-  Maximum Paths (i),
+MAX_CLIENT_PATHS Frame {
+  Type (i) = 0x15228c0c,
+  Maximum Number of Client Paths (i),
 }
-~~~
-{: #fig-max-paths-frame-format title="MAX_PATHS Frame Format"}
-
-MAX_PATHS frames contain the following field:
 
-Maximum Path Identifier:
-: A count of the cumulative number of path that can be opened 
-over the lifetime of the connection. This value cannot exceed 2^32-1, as it is not 
-possible to encode Path IDs larger than 2^32-1. Receipt of a frame that permits 
-opening of a path with Path Identifier larger than this limit MUST be treated 
-as a connection error of type FRAME_ENCODING_ERROR.
-
-Loss or reordering can cause an endpoint to receive a MAX_PATHS frame with 
-a lower path limit than was previously received. MAX_PATHS frames that 
-do not increase the path limit MUST be ignored.
+MAX_SERVER_PATHS Frame {
+  Type (i) = 0x15228c0d,
+   Maximum Number of Server Paths (i),
+}
 
-An endpoint MUST NOT initiate a path with a path ID higher than the Maximum Paths value.
-An endpoint MUST terminate the a connection with an error of type MP_PROTOCOL_VIOLATION if a peer opens more paths than was permitted. 
+~~~
+{: #fig-max-paths-frame-format title="MAX_PATHS Frame Format"}
 
+MAX_CLIENT_PATHS and MAX_SERVER_PATHS frames contain the following field:
+
+Maximum Number of Client/Server Path:
+: A count of the cumulative number of respectively client orserver path that can be opened
+over the lifetime of the connection. The client path identifiers sent by either
+endpoint MUST be lower than 2 times the Maximum Number of Client Paths announced py the peer.
+The server path identifiers sent by either
+endpoint MUST be lower than 2 times the Maximum Number of Server Paths announced py the peer
+plus 1. These values cannot exceed 2^30-1, as it is not
```

Comment:
This should be 2^31 - 1, right? (2^31 - 1) * 2 = 2^32 - 2 (max client-initiated path ID) and (2^31 - 1) * 2 + 1 =  2^32 - 1 (max client-server path ID)
```suggestion
plus 1. These values cannot exceed 2^31-1, as it is not
```